src/components/DataFilter/index.js

```diff
@@ -0,0 +1,134 @@
+import { Grid, Typography, ButtonGroup, Button } from "@material-ui/core";
```

Comment:
This should probably be named `SourcesFilter` since that's what it's filtering, right?